### PR TITLE
v1.0.3 - added more cats

### LINKED ISSUE #5: Add more cats!
I've found more cats at https://github.com/audy/catsay/tree/master/cats and https://github.com/paulkaefer/cowsay-files/tree/master/cows :-D

### PR DESCRIPTION
**Pull request by wiiiimm**
fixed #5

### CODE REVIEW

File: cats.json [hunk 1]
```diff
@@ -1,169 +1,665 @@
...
+  "                           |  /\\/\\  |",
+  "                          .'___  ___`.",
+  "                         /  \\|/  \\|/  \\",
+  "        _.--------------( ____ __ _____)",
+  "     .-' \\  -. | | | | | \\ ----\\/---- /",
+  "   .'\\  | | / \\` | | | |  `.  -'`-  .'",
+  "  /`  ` ` '/ / \\ | | | | \\  `------'\\",
+  " /-  `-------.' `-----.       -----. `---.",
+  "(  / | | | |  )/ | | | )/ | | | | | ) | | )",
+  " `._________.'_____,,,/\\_______,,,,/_,,,,/"
```
Comment (piranna) [inline cats.json]: Can you restore the indentation?

File: cats.json [hunk 2]
```diff
@@ -1,169 +1,665 @@
...
+  "          -MM        MMMMMMM",
+  "          ^  M+     MMMMMMMMM",
+  "              MMMMMMM MM MM MM",
+  "                   MM MM MM MM",
+  "                   MM MM MM MM",
+  "                .~~MM~MM~MM~MM~~.",
+  "             ~~~~MM:~MM~~~MM~:MM~~~~",
+  "            ~~~~~~==~==~~~==~==~~~~~~",
+  "             ~~~~~~==~==~==~==~~~~~~",
+  "                 :~==~==~==~==~~"
```
Comment (piranna) [inline cats.json]: Remove leading whitespaces in the image, left only the bare minimum ones.

File: cats.json [hunk 3]
```diff
@@ -1,169 +1,665 @@
...
+  ],
+  "ready-to-attack":
+  [
+  "          _._     _,-'\"\"`-._",
+  "     (,-.`._,'(       |\\`-/|",
+  "         `-.-' \\ )-`( , o o)",
+  "               `-    \\`_`\"'-"
```
Comment (piranna) [inline cats.json]: Same here.

File: cats.json [hunk 4]
```diff
@@ -1,169 +1,665 @@
...
+  "    　∧_∧",
+  "　　 (　･ω･)　",
+  "　 ＿|　⊃／(＿＿_",
+  "／　└-(＿＿＿_／"
+  ],
+  "sleepy": [
+  "      |\\      _,,,---,,_",
+  "      /,`.-'`'    -.  ;-;;,_",
+  "     |,4-  ) )-,_..;\\ (  `'-'",
+  "    '---''(_/--'  `-'\\_)"
```
Comment (piranna) [inline cats.json]: Same here.

File: cats.json [hunk 5]
```diff
@@ -1,169 +1,665 @@
...
+  "　| 丿 ＼＼ ",
+  "　| |　　)/",
+  "ノ )　　 Lﾉ"
+  ],
+  "hector": [
+  "    +++",
+  "   //             ",
+  "  ||________^__^",
+  "  |     ~=<(@ .@)>=~",
+  "  |___m_____m__m"
```
Comment (piranna) [inline cats.json]: Same here.

File: cats.json [hunk 6]
```diff
@@ -1,169 +1,665 @@
...
+  "|　　|",
+  "|　　|",
+  "|　　|",
+  "|　　|",
+  "Ｕ\"Ｕ"
+  ],
+  "mini": [
+  "  /\\_/\\",
+  " ( o.o )",
+  "  m   m  "
```
Comment (piranna) [inline cats.json]: Same here.

File: cats.json [hunk 7]
```diff
@@ -1,169 +1,665 @@
...
+  " ( | | )",
+  "(__d b__)"
+  ],
+  "lion": [
+  "      /\";;:;;\"\\",
+  "    (:;/\\,-,/\\;;)",
+  "   (:;{  d b  }:;)",
+  "    (:;\\__Y__/;;)-----------,,_",
+  "     ,..\\  ,..\\      ___/___)__`\\",
+  "    (,,,)~(,,,)`-._##____________)"
```
Comment (piranna) [inline cats.json]: Same here.

File: cats.json [hunk 8]
```diff
@@ -1,169 +1,665 @@
...
+  "       ; *` _.   `*-._          ",
+  "       `-.-'          `-.       ",
+  "         ;       `       `.     ",
+  "         :.       .        \\    ",
+  "         . \\  .   :   .-'   .   ",
+  "         '  `+.;  ;  '      :   ",
+  "         :  '  |    ;       ;-. ",
+  "         ; '   : :`-:     _.`* ;",
+  "[bug] .*' /  .*' ; .*`- +'  `*' ",
+  "      `*-*   `*-*  `*-*'"
```
Comment (piranna) [inline cats.json]: Maybe remove the bug and left the cat alone?

File: cats.json [hunk 9]
```diff
@@ -1,169 +1,665 @@
...
+  "                 __-----^----|   |----^--#----+",
+  "                H            |   |             #",
+  "              /             /     \\             \\",
+  "            {          /       *                 \\",
+  "            -/       /                 9\\         \\-",
+  "            {        |                   |        }",
+  "-Ling Yang- {        }                   {        }",
+  "            #   ^    }                   {        }",
+  "            {        }-\\__---\\____-----#-{        }",
+  "           {^_^} v{-_-}                 {*_*} v(0_0}"
```
Comment (piranna) [inline cats.json]: Left the cat alone, without the name.

File: cats.json [hunk 10]
```diff
@@ -1,169 +1,665 @@
...
+    "         MMMM:HHHHMM:HHHHM:MHHMIMMMHHHIHHM:MMMM.",
+    "         :MMM:IHHHMM:HHHMM:MHHMIIMMMHHMM:MMMMMMM:",
+    "         :MMM:IHHHM:HHHHMM:MMHHHIHHMMM:MMMMMMMMMM",
+    "          MHM:IHHHM:HHHMMM:MMHHHHIIIMMIIMMMMMMMMM",
+    "          MHM:HHHHM:HHHMMM:HMMHHHHHHHHHMMMMMMMMM:",
+    "       .MI:MM:MHHMM:MHMMHMHHMMMMHHHHHHHMMMMMMMMM'",
+    "      :IM:MMIM:M:MM:MH:MM:MH:MMMMMHHHHHMMMMMMMM'",
+    "      :IM:M:IM:M:HM:IMIHM:IMI:MMMMMHHHMMMMMM:'",
+    "       'M:MHM:HM:MN:HMIHM::M'   '::MMMMMMM:'",
+    "          'M'HMM'M''M''HM'I'"
```
Comment (piranna) [inline cats.json:600]: Same here.

File: cats.json [hunk 11]
```diff
@@ -1,169 +1,665 @@
...
+    "|      `. /  '       ))    (   )  \\>     |     |",
+    "|        ;   |      ((      \\ /    \\___  |     |",
+    "|        ;   |      _))      `'.-'. ,-'` '     |",
+    "|        |    `.   ((`            |/    /      |",
+    "|        \\     ).  .))            '    .       |",
+    "|     ----`-'-'  `''.::.________:::mx'' ---    |",
+    "|                                              |",
+    "|                                              |",
+    "|                                              |",
+    "'=============================================='"
```
Comment (piranna) [inline cats.json]: Remove border.

File: cats.json [hunk 12]
```diff
@@ -1,169 +1,665 @@
...
+    " /   |   \\..   ` ./ `.   _,_  ,'  ``'  /'   :'|      _,''/       ",
+    "/   /'. :   \\.   _    [_]   `[_]  .__,,|   _....,--=/'  /:       ",
+    "|   \\_| :    `.-' `.    _.._     /     . ,'  :. ':/'  /'  `.     ",
+    "`.   '`'`.         `. ,.'   ` .,'     :'/ ':..':.    |  .:' `.   ",
+    "  \\.      \\          '               :' |    ''''      ''     `. ",
+    "    `''.   `|        ':     .      .:' ,|         .  ..':.      |",
+    "      /'   / '\"-..._  :   .:'    _;:.,'  \\.     .:'   :. ''.    |",
+    "     (._,.'        '`''''''''''''          \\.._.:      ':  ':   /",
+    "________                                      '`- ._    ,:__,,-' ",
+    "  |ooShy                                            ``''"
```
Comment (piranna) [inline cats.json]: Remove name.

**Review by piranna — CHANGES_REQUESTED** [review]

File: cats.json (hunk 8 above)
Comment (wiiiimm) [reply cats.json]: lol.

**Comment by wiiiimm** [comment]
@coleww it's ready. the world **needs** these cats... please release them to the public!

**Comment by coleww** [comment]
sorry, but i am not maintaining this package anymore.

**Pull request closed without merging**